src/pages/regulatory/index.tsx

```diff
@@ -110,6 +110,8 @@ const StyledLinkText = styled(LinkText)`
     font-weight: bold;
 `
 
+const svg_address = '_t_Deriv (SVG) LLC is located in 1st Floor SVG Teachers Credit Union Limited, Uptown Building, Cnr. Middle & James Street, Kingstown, St Vincent and the Grenadines (company no. 273 LLC 2020)._t_'
```

Comment:
Hi @eduard-deriv I'm just curious why we are putting the address inside a constant and not update it inside the `localize` itself ? 🤔 🙏

Reply:
it doesn’t matter, we can do it inside `localize`, it seemed to me that this text was so long and I put it in a separate variable.